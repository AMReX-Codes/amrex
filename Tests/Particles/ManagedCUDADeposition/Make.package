CEXE_headers += MyParticleContainer.H CUDAManagedAllocator.H
CEXE_sources += main.cpp MyParticleContainer.cpp solve_for_accel.cpp solve_with_f90.cpp

FEXE_headers += deposit_F.H
f90EXE_sources += deposit_$(DIM)d.f90

ifeq ($(USE_CUDA), TRUE)
f90EXE_sources += cuda_deposit_$(DIM)d.f90
endif
