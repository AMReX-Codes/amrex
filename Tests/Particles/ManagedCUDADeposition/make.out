Loading ../../..//Tools/GNUMake/comps/pgi.mak...
Loading ../../..//Tools/GNUMake/sites/Make.ccse...
Depending ../../..//Src/Particle/AMReX_TracerParticles.cpp ...
Depending MyParticleContainer.cpp ...
Depending main.cpp ...
Loading ../../..//Tools/GNUMake/comps/pgi.mak...
Loading ../../..//Tools/GNUMake/sites/Make.ccse...
Compiling main.cpp ...
pgc++  -gopt -fast --c++11 -Mcuda=cuda8.0  -noacc -DNDEBUG -DBL_TINY_PROFILING -DAMREX_GIT_VERSION=\"17.06-290-g5c9520666235-dirty\" -DCUDA -DCUDA_UM -DBL_COALESCE_FABS -DDEVICE_LAUNCH_PROCEDURE='attributes(global) &' -DDEVICE_PROCEDURE='attributes(device) &' -DBL_SPACEDIM=3 -DAMREX_SPACEDIM=3 -DBL_FORT_USE_UNDERSCORE -DBL_Linux -DUSE_PARTICLES -DPARTICLES -DMG_USE_FBOXLIB -DBL_USE_F_BASELIB -I. -I/usr/local/cuda-8.0//include -I../../..//Src/Base -I../../..//Src/Boundary -I../../..//Src/AmrCore -I../../..//Src/Particle -I../../..//Src/LinearSolvers/C_to_F_MG -I../../..//Src/LinearSolvers/C_CellMG -I../../..//Src/LinearSolvers/F_MG -I../../..//Src/F_BaseLib -c main.cpp -o tmp_build_dir/o/3d.pgi.TPROF.CUDA.EXE/main.o
Compiling MyParticleContainer.cpp ...
pgc++  -gopt -fast --c++11 -Mcuda=cuda8.0  -noacc -DNDEBUG -DBL_TINY_PROFILING -DAMREX_GIT_VERSION=\"17.06-290-g5c9520666235-dirty\" -DCUDA -DCUDA_UM -DBL_COALESCE_FABS -DDEVICE_LAUNCH_PROCEDURE='attributes(global) &' -DDEVICE_PROCEDURE='attributes(device) &' -DBL_SPACEDIM=3 -DAMREX_SPACEDIM=3 -DBL_FORT_USE_UNDERSCORE -DBL_Linux -DUSE_PARTICLES -DPARTICLES -DMG_USE_FBOXLIB -DBL_USE_F_BASELIB -I. -I/usr/local/cuda-8.0//include -I../../..//Src/Base -I../../..//Src/Boundary -I../../..//Src/AmrCore -I../../..//Src/Particle -I../../..//Src/LinearSolvers/C_to_F_MG -I../../..//Src/LinearSolvers/C_CellMG -I../../..//Src/LinearSolvers/F_MG -I../../..//Src/F_BaseLib -c MyParticleContainer.cpp -o tmp_build_dir/o/3d.pgi.TPROF.CUDA.EXE/MyParticleContainer.o
Compiling AMReX_TracerParticles.cpp ...
pgc++  -gopt -fast --c++11 -Mcuda=cuda8.0  -noacc -DNDEBUG -DBL_TINY_PROFILING -DAMREX_GIT_VERSION=\"17.06-290-g5c9520666235-dirty\" -DCUDA -DCUDA_UM -DBL_COALESCE_FABS -DDEVICE_LAUNCH_PROCEDURE='attributes(global) &' -DDEVICE_PROCEDURE='attributes(device) &' -DBL_SPACEDIM=3 -DAMREX_SPACEDIM=3 -DBL_FORT_USE_UNDERSCORE -DBL_Linux -DUSE_PARTICLES -DPARTICLES -DMG_USE_FBOXLIB -DBL_USE_F_BASELIB -I. -I/usr/local/cuda-8.0//include -I../../..//Src/Base -I../../..//Src/Boundary -I../../..//Src/AmrCore -I../../..//Src/Particle -I../../..//Src/LinearSolvers/C_to_F_MG -I../../..//Src/LinearSolvers/C_CellMG -I../../..//Src/LinearSolvers/F_MG -I../../..//Src/F_BaseLib -c ../../..//Src/Particle/AMReX_TracerParticles.cpp -o tmp_build_dir/o/3d.pgi.TPROF.CUDA.EXE/AMReX_TracerParticles.o
make: *** No rule to make target `tmp_build_dir/o/3d.pgi.TPROF.CUDA.EXE/AMReX_CUDAManagedAllocator.H', needed by `main3d.pgi.TPROF.CUDA.ex'.  Stop.
